panic: runtime error: invalid memory address or nil pointer dereference when pulling large object

`go version go1.8 linux/386`
`git-lfs/2.3.4 (GitHub; linux 386; go 1.8; git 6f503b2)`

Built by cloning the git-lfs repo and running the following commands:
```
git clone https://github.com/github/git-lfs.git
cd git-lfs
./script/bootstrap
```

I'm getting the following error pulling large files via Bamboo's Git LFS feature:
```
Fetching 'refs/heads/somebranch' from 'ssh://git@git.server.com/blah/repo.git'. Will try to do a shallow fetch.
Warning: Permanently added '[127.0.0.1]:34441' (RSA) to the list of known hosts.
fetch: Fetching reference refs/heads/somebranch
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x804965c]

goroutine 1 [running]:
sync/atomic.AddUint64(0x18dfd2ec, 0x7061358, 0x0, 0x0, 0x0)
        /usr/local/go/src/sync/atomic/asm_386.s:112 +0xc
github.com/git-lfs/git-lfs/tq.(*Meter).Add(0x18dfd2d0, 0x7061358, 0x0)
        /home/build/git-lfs/src/github.com/git-lfs/git-lfs/tq/meter.go:114 +0x7a
github.com/git-lfs/git-lfs/commands.readyAndMissingPointers(0x189d9b40, 0x6, 0x8, 0x18a3a3c0, 0x189d8000, 0x1891a068, 0x0, 0x0, 0x0, 0x0, ...)
        /home/build/git-lfs/src/github.com/git-lfs/git-lfs/commands/command_fetch.go:351 +0x398
github.com/git-lfs/git-lfs/commands.fetchAndReportToChan(0x189d9b40, 0x6, 0x8, 0x18a3a3c0, 0x0, 0x8)
        /home/build/git-lfs/src/github.com/git-lfs/git-lfs/commands/command_fetch.go:271 +0x50
github.com/git-lfs/git-lfs/commands.fetchRef(0x18a08070, 0x28, 0x18a3a3c0, 0x1)
        /home/build/git-lfs/src/github.com/git-lfs/git-lfs/commands/command_fetch.go:142 +0x79
github.com/git-lfs/git-lfs/commands.fetchCommand(0x18972820, 0x189c9040, 0x2, 0x2)
        /home/build/git-lfs/src/github.com/git-lfs/git-lfs/commands/command_fetch.go:87 +0x208
github.com/git-lfs/git-lfs/vendor/github.com/spf13/cobra.(*Command).execute(0x18972820, 0x189c8f60, 0x2, 0x2, 0x18972820, 0x189c8f60)
        /home/build/git-lfs/src/github.com/git-lfs/git-lfs/vendor/github.com/spf13/cobra/command.go:477 +0xf8
github.com/git-lfs/git-lfs/vendor/github.com/spf13/cobra.(*Command).Execute(0x18972270, 0x18943f94, 0x1)
        /home/build/git-lfs/src/github.com/git-lfs/git-lfs/vendor/github.com/spf13/cobra/command.go:551 +0x278
github.com/git-lfs/git-lfs/commands.Run()
        /home/build/git-lfs/src/github.com/git-lfs/git-lfs/commands/run.go:71 +0x194
main.main()
        /home/build/git-lfs/git-lfs.go:35 +0xf8
```

--- a/git/odb/object_db.go
+++ b/git/odb/object_db.go
@@ -16,16 +16,19 @@ import (
 // ObjectDatabase enables the reading and writing of objects against a storage
 // backend.
 type ObjectDatabase struct {
-	// s is the storage backend which opens/creates/reads/writes.
-	s storer
-	// packs are the set of packfiles which contain all packed objects
-	// within this repository.
-	packs *pack.Set
+	// members managed via sync/atomic must be aligned at the top of this
+	// structure (see: https://github.com/git-lfs/git-lfs/pull/2880).
 
 	// closed is a uint32 managed by sync/atomic's <X>Uint32 methods. It
 	// yields a value of 0 if the *ObjectDatabase it is stored upon is open,
 	// and a value of 1 if it is closed.
 	closed uint32
+
+	// s is the storage backend which opens/creates/reads/writes.
+	s storer
+	// packs are the set of packfiles which contain all packed objects
+	// within this repository.
+	packs *pack.Set
 
 	// temp directory, defaults to os.TempDir
 	tmp string

--- a/git/odb/object_writer.go
+++ b/git/odb/object_writer.go
@@ -13,14 +13,17 @@ import (
 // writes data given to it, and keeps track of the SHA1 hash of the data as it
 // is written.
 type ObjectWriter struct {
-	// w is the underling writer that this ObjectWriter is writing to.
-	w io.Writer
-	// sum is the in-progress hash calculation.
-	sum hash.Hash
+	// members managed via sync/atomic must be aligned at the top of this
+	// structure (see: https://github.com/git-lfs/git-lfs/pull/2880).
 
 	// wroteHeader is a uint32 managed by the sync/atomic package. It is 1
 	// if the header was written, and 0 otherwise.
 	wroteHeader uint32
+
+	// w is the underling writer that this ObjectWriter is writing to.
+	w io.Writer
+	// sum is the in-progress hash calculation.
+	sum hash.Hash
 
 	// closeFn supplies an optional function that, when called, frees an
 	// resources (open files, memory, etc) held by this instance of the

--- a/lfsapi/stats.go
+++ b/lfsapi/stats.go
@@ -16,9 +16,9 @@ import (
 )
 
 type httpTransfer struct {
-	URL             string
-	Method          string
-	Key             string
+	// members managed via sync/atomic must be aligned at the top of this
+	// structure (see: https://github.com/git-lfs/git-lfs/pull/2880).
+
 	RequestBodySize int64
 	Start           int64
 	ResponseStart   int64
@@ -28,6 +28,9 @@ type httpTransfer struct {
 	DNSEnd          int64
 	TLSStart        int64
 	TLSEnd          int64
+	URL             string
+	Method          string
+	Key             string
 }
 
 type statsContextKey string

--- a/tasklog/percentage_task.go
+++ b/tasklog/percentage_task.go
@@ -10,13 +10,16 @@ import (
 // PercentageTask is a task that is performed against a known number of
 // elements.
 type PercentageTask struct {
-	// msg is the task message.
-	msg string
+	// members managed via sync/atomic must be aligned at the top of this
+	// structure (see: https://github.com/git-lfs/git-lfs/pull/2880).
+
 	// n is the number of elements whose work has been completed. It is
 	// managed sync/atomic.
 	n uint64
 	// total is the total number of elements to execute work upon.
 	total uint64
+	// msg is the task message.
+	msg string
 	// ch is a channel which is written to when the task state changes and
 	// is closed when the task is completed.
 	ch chan *Update

